### PR TITLE
[CONTENT] As You Like It-inspired Patrol

### PR DESCRIPTION
**Pull request by tobes03**
## About The Pull Request

Another patrol where two cats on a border patrol find two kittypets in an argument.

## Why This Is Good For ClanGen

fun!

## Proof of Testing
image
image
image
image

### CODE REVIEW

File: resources/dicts/patrols/general/border.json [hunk 1]
```diff
@@ -3001,5 +3001,522 @@
...
+                ["meeting", "kittypet", "age:young adult"],
+                ["meeting", "kittypet", "age:young adult"]
+                ],
+                "art": "gen_bord_twocat_hostile_loner_loner"
+        }
+	],
+
+	"antag_fail_outcomes": [
+		{
+            "text": "p_l doesn't care about kittypet drama. {PRONOUN/p_l/subject/CAP} {VERB/p_l/interrupt/interrupts} and {VERB/p_l/tell/tells} the kittypets to get lost. One kittypet doesn't appreciate p_l's tone and takes a swipe at {PRONOUN/p_l/object}. r_c jumps into help a humiliated p_l drive off the kittypets.",
```
Comment (scribblecrumb) [inline resources/dicts/patrols/general/border.json]:
```suggestion
            "text": "p_l doesn't care about kittypet drama. {PRONOUN/p_l/subject/CAP} {VERB/p_l/interrupt/interrupts} and {VERB/p_l/tell/tells} the kittypets to get lost. One kittypet doesn't appreciate p_l's tone and takes a swipe at {PRONOUN/p_l/object}. r_c jumps in to help a humiliated p_l drive off the kittypets.",

```

**Review by scribblecrumb — CHANGES_REQUESTED** [review]
loving this!  I think it might also be fun to add the "exists" tag to the new cats.  I like the idea of linking two cats that the player might already be aware of.

**Comment by tobes03** [comment]
ooh fun!! I wasn't really sure how the "exists" tag worked but if it'd be okay here that would be great. I'll add it :)

**Comment by scribblecrumb** [comment]
It should work here!  It'll only trigger if there's existing cats of the correct status and age

**Review by scribblecrumb — APPROVED** [review]

**Pull request merged**